src/other/broadcasting.jl

```diff
@@ -129,7 +129,7 @@ if isdefined(Base, :dotgetproperty)
         if columnindex(df, col) == 0
             return LazyNewColDataFrame(df, Symbol(col))
         else
-            Base.depwarn("In the future this operation will allocate a new column" *
+            Base.depwarn("In the future this operation will allocate a new column " *
                     "instead of performing an in-place assignment.", :dotgetproperty)
```

Comment:
```suggestion
            Base.depwarn("In the future this operation will allocate a new column " *
                         "instead of performing an in-place assignment.", :dotgetproperty)
```